pdr_backend/pdr_dashboard/util/format.py

```diff
@@ -0,0 +1,108 @@
+from typing import Union
+
+from enforce_typing import enforce_types
+from numerize import numerize
+
+
+@enforce_types
+def format_metric(metric: Union[int, float], value_id: str) -> str:
+    match value_id:
+        case "feeds_page_Accuracy_metric" | "accuracy_metric":
```

Comment:
I would also add another parameter metric_type to replace value Id. i.e. instead of looping through the ids, we loop through an accepted list of format keys, i.e. value, stake, cost.
This means we can reuse these in table cols.